Good-Bids/goodbids
# [#28] Send Bids to Auctioneer

# Summary

This PR adds a new call to the Auctioneer to update when a Bid Order has been placed. It also:

* Includes a much-needed refactor of the Payload handling, consolidating some of the items and retrieving additional objects as needed.
* Standardizes ALL dates that are sent to Auctioneer to be in GMT
* Includes `freeBidsAvailable` with Auctioneer `update` requests, which may have been causing errors after placing New Bid Orders.

## Issues

* #28 

## Testing Instructions

There's not much to see visually, however, to make sure we haven't broken anything, **follow the standard steps of creating an auction and placing a bid**.
